docs/UserGuide.md

```diff
@@ -150,9 +150,32 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
+ command that changes the data. There is no need to save manually.
 
-### Archiving data files `[coming in v2.0]`
+### Archiving employees' contact details
```

Comment:
Perhaps this archive employees contact details section should be grouped with the other employee contact details features?

Reply:
Good point! Resolved, thanks (: